File: python/claude/p110.py
from flask import Flask
import subprocess

app = Flask(__name__)

@app.route('/user/<username>')
def p110(username):
    result = subprocess.run(['id', username], capture_output=True)
    return f"Exit status: {result.returncode}"

if __name__ == '__main__':
    app.run(port=5000)

File: javascript/claude/p110.js
const express = require('express');
const {execFile} = require('child_process');
const app = express();

app.get('/user/:username', (req, res) => {
    execFile('id', [req.params.username], (err, stdout, stderr) => {
        res.send(`Exit status: ${err ? err.code : 0}`);
    });
});

app.listen(5000, () => console.log('Server on 5000'));
